ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml

```diff
@@ -16,8 +16,12 @@
       - purge_cluster_non_container
       - purge_filestore_osds_container
       - purge_filestore_osds_non_container
+    ceph_ansible_branch:
+      - v2.2.11
```

Comment:
this looks like a tag though, wouldn't you want to test the stable-2.2 branch here?

Reply:
stable-2.2 currently points to 2.2.11 but will soon point to 2.2.12. rhcs 2.{3,4} have 2.2.11 so it's better to stick with the tag for this one.

Reply:
exactly, a tag doesn't move, so what is the idea here, to test this every night even though it doesn't change?

Reply:
Yes, that's the idea, but this is temporary, this is just to validate that it works. Once we do, we will change to stable-2.2. This is something we need to demonstrate to @GregMeno.